vaadin-spring-tests/test-spring-security-fusion/frontend/views/login-view.ts

```diff
@@ -22,7 +22,7 @@ export class LoginView extends View implements AfterEnterObserver {
 
     // As we do not know if the target is a resource or a Fusion view or a Flow view, we cannot just use Router.go
     window.location.href =
-      result.redirectUrl || this.returnUrl || result.defaultUrl || "/";
```

Comment:
Why shouldn't `defaultUrl` be used?

Reply:
The `defaultUrl`(which is determined by Spring Security) is `/`, which doesn't work with the context path.

Reply:
Made a ticket for the issue https://github.com/vaadin/fusion/issues/33, let's merge this PR first, I have another PR https://github.com/vaadin/spring/pull/839 depends on this one.